tests/modules/nf-core/fastqc/main.nf.test

```diff
@@ -0,0 +1,32 @@
+nextflow_process {
+
+    name "Test Process FASTQC"
+    script "modules/nf-core/fastqc/main.nf"
+    process "FASTQC"
+    tag "fastqc"
+
+    test("Should run without failures") {
```

Comment:
```suggestion
    test("test_fastqc_single_end") {
```

Reply:
Good catch, but I think we can do away with the _'s now. Thoughts? I had it actually just as "Single End", I guess I regenerated it?

Reply:
yup...that works too